### ISSUE TITLE
May not be needed

### ISSUE THREAD

**Issue by timroster** [cloud-native-toolkit/terraform-gitops-terraform-ibm-ocp-proxyconfig#1]
After filing the request - the proxy module code was brought in to: https://github.com/cloud-native-toolkit/terraform-ocp-proxyconfig

Oh - and the initial request was for a straight terraform, not gitops module: https://github.com/cloud-native-toolkit/software-everywhere/issues/353

However, it might be possible to rework what is being done in the terraform-based module into the gitops style. However if ArgoCD is to inherit the cluster-wide proxy, it probably needs to be launched after, and not before the cluster-wide proxy is created.

**Comment by triceam** [comment]
ok, so do you want me to remove that module, and we can revisit later?

**Comment by timroster** [comment]
yes - that would be great. thx. I should have closed the issue when the other repo was created.
